core/interop/Streams/readme.md

```diff
@@ -0,0 +1,37 @@
+# Interop with .NET Stream sample
+
+The [`Stream`](https://learn.microsoft.com/dotnet/api/system.io.stream) data type is one of the most common non-trivial data types for interop. This sample provides two distinct mechanism for working with the `Stream` type in interop scenarios.
+
+1) The projection of the `Stream` type using C-style exports.
+
+1) The projection of the `Stream` type using the COM [`IStream`](https://learn.microsoft.com/windows/win32/api/objidl/nn-objidl-istream) interface. This approach utilizes [`ComWrappers`](https://learn.microsoft.com/dotnet/api/system.runtime.interopservices.comwrappers) and the COM source generator.
```

Comment:
I know we don't have COM source generator docs yet - do we want to just link to the GeneratedComInterface API doc for now